app/webpacker/scripts/autocomplete.js

```diff
@@ -32,7 +32,7 @@ const initAutocomplete = ($el, $input) => {
   const el = document.getElementById($el);
   const inputValueTemplate = result => (typeof result === "string" ? result : result && result.name);
   const suggestionTemplate = result =>
-    typeof result === "string" ? result : result && `${result.name} (${result.providerCode})`;
+    typeof result === "string" ? result : result && `${result.name} (${result.code})`;
```

Comment:
Does this change break the provide code?

Reply:
It does the opposite. It unbreaks it 😄 ... The code highlighted in red is C# . I'll add as a separate commit